doc/en/user/source/services/internationalization/index.rst

```diff
@@ -31,6 +31,10 @@ The response content language can be selected using the ``AcceptLanguages`` requ
 
 * The internationalized elements will be titles, abstracts and keywords.
 
+* If no language is defined after checking the i18n box, the text will be attached to a no Locale, which is the default language.
+
+* If no language code is specified, text value with default language will be returned.
```

Comment:
I would remove these entries here and add the explanation in the `Default Language `section down here.
Should be made clear that: from UI it is possible to set a i18n content with null language and that that content will be used in case no default language was specified from Service/Global settings.
Moreover the part telling that GeoServer will try to use the server default locale should go since now the `GeoServerDefaultLocale `will instead return null.

Reply:
Done.